# Requires the `data-test-id` attribute on MUI <Radio> elements.

## Rule Details

Examples of **incorrect** code for this rule:

```html
<Radio value="value" />
```

Examples of **correct** code for this rule:

```html
<Radio value="value" data-test-id="testRadio" />
```

